website/src/constants/defaults.js

```diff
@@ -1,8 +1,8 @@
 export const MAPBOX_STYLES = {
-  LIGHT: 'mapbox://styles/uberdata/cive48w2e001a2imn5mcu2vrs',
-  DARK: 'mapbox://styles/uberdata/cive485h000192imn6c6cc8fc',
+  LIGHT: 'https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json',
+  DARK: 'https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json',
```

Comment:
These styles are not the same color scheme as the ones we have today.

Reply:
@Josmorsot let's take the styles of `mapbox://styles/uberdata/cive48w2e001a2imn5mcu2vrs` and change the source to our tile server.

Reply:
And place the files at `website/static`

Reply:
Done: 8143721